atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/impl/AtlasDbServicesImpl.java

```diff
@@ -39,18 +42,51 @@
 
     private SerializableTransactionManager tm;
 
-    public static AtlasDbServices connect(String configFileName) throws IOException {
+    public static AtlasDbServices connect(File configFile, String configRoot) throws IOException {
         ObjectMapper configMapper = Jackson.newObjectMapper(new YAMLFactory());
-        AtlasDbServerConfiguration config = configMapper.readValue(new File(configFileName), AtlasDbServerConfiguration.class);
-        SerializableTransactionManager tm = TransactionManagers.create(config.getConfig(), Optional.<SSLSocketFactory>absent(), ImmutableSet.<Schema>of(),
+        JsonNode node = getConfigNode(configMapper, configFile, configRoot);
+        AtlasDbServerConfiguration config = configMapper.treeToValue(node, AtlasDbServerConfiguration.class);
+        SerializableTransactionManager tm = TransactionManagers.create(
+                config.getConfig(),
+                Optional.<SSLSocketFactory>absent(),
+                ImmutableSet.<Schema>of(),
                 new TransactionManagers.Environment() {
                     @Override
                     public void register(Object resource) {
                     }
-                }, false);
+                },
+                true);
```

Comment:
I think I just wasn't paying attention.  I think we might as well allow the clis to mess with hidden tables?  Any objection?